cfme/tests/infrastructure/test_host.py

```diff
@@ -162,3 +180,22 @@ def test_tag_host_after_provider_delete(provider, appliance, setup_provider, req
     provider.wait_for_delete()
     tag = host.add_tag()
     request.addfinalizer(lambda: host.remove_tag(tag))
+
+
+def test_250_vmware_hosts_loading(appliance, create_250_hosts):
```

Comment:
**Question** Does this test only apply to CFME 5.10? Or has it been backported to 5.9z stream?

Reply:
This test applies to both streams